vipps-recurring-api-faq.md

```diff
@@ -39,6 +41,15 @@ Document version: 1.5.1.
 * [Invoicing](#invoicing)
 * [Questions?](#questions)
 
+## Does Vipps automatically create charges for an agreement?
+ 
+ No. Vipps does _not_ create charges based on the agreement, this is left up to the merchant to create.  
+ When a merchant creates a charge, Vipps will actually attempt to charge the customer.  
+ If the merchant doesn't create a charge, nothing will be done.
+ ## Do I need to store card data?
+
+ No. Vipps all payment details.
```

Comment:
Missing word?

Reply:
Removed a word too many. It was supposed to have "handled" in there. Fixed and pushed.